samples/django_sample/settings.py

```diff
@@ -66,12 +66,12 @@
 
 ROOT_URLCONF = 'django_sample.urls'
 
-TEMPLATE_DIRS = (
+TEMPLATE_DIRS = [
     # Put strings here, like "/home/html/django_templates"
     # Always use forward slashes, even on Windows.
     # Don't forget to use absolute paths, not relative paths.
     os.path.join(os.path.dirname(__file__), 'templates')
```

Comment:
Rather than change `TEMPLATE_DIRS` to a list, shouldn't a comma simply be added to the end of its one element?